src/class-rokka-helper.php

```diff
@@ -644,6 +660,24 @@ public function get_stacks_to_sync() {
 	}
 
 	/**
+	 * Checks if the autoformat option has changed since the last stack synchronization.
+	 *
+	 * @param \Rokka\Client\Core\Stack $stack Stack to check option.
+	 *
+	 * @return bool
+	 */
+	protected function autoformat_changed( $stack ) {
```

Comment:
Not sure if I am missing context, but this function could be generalized for plugin-options? Now it can only regard changes for autoformat ( which is probably all that is needed for now ) but generalizing it could prevent duplicated code in future and it seems likely that this will be needed. What do you think?